GameDefinition/Content/RuleKind.hs

```diff
@@ -46,4 +46,22 @@ standardRules = RuleContent
   , rnearby = 20
   , rstairWordCarried = ["staircase"]  -- only one, so inert
   , rsymbolProjectile = '|'
+  , rsymbolLight      = '('
+  , rsymbolTool       = '('
+  , rsymbolSpecial    = '*'  -- don't overuse, because it clashes with projectiles
+  , rsymbolGold       = '$'
+  , rsymbolNecklace   = '"'
+  , rsymbolRing       = '='
+  , rsymbolPotion     = '+'  -- concoction, bottle, jar, vial, canister
```

Comment:
```suggestion
  , rsymbolPotion     = '!'  -- concoction, bottle, jar, vial, canister
```

Reply:
The clash with door symbol may be why CI failed, but I can't say for sure, because Travis is still down and AppVeyor still has broken test failure output.